db/migrate/20230830014805_add_spec_sha256_to_version.rb

```diff
@@ -0,0 +1,5 @@
+class AddSpecSha256ToVersion < ActiveRecord::Migration[7.0]
+  def change
+    add_column :versions, :spec_sha256, :string
```

Comment:
Maybe a length? It's always 44 bytes, if I'm not mistaken.

Reply:
can do! it just isnt there for the gem sha256 column, iirc.